create-container.sh

```diff
@@ -120,6 +120,10 @@ sudo lxc-attach -n "$NAME" -- /bin/sh -c "/usr/bin/id -u $DEVENV_USER || /usr/sb
 echo "Copying system user's SSH public key to $DEVENV_USER user in container"
 sudo lxc-attach -n "$NAME" -- sudo -u "$DEVENV_USER" -- sh -c "/bin/mkdir -p /home/$DEVENV_USER/.ssh && echo $ssh_key > /home/$DEVENV_USER/.ssh/authorized_keys"
 
+# Debian Stretch Sudo install
+if [ $DISTRIBUTION == "debian" ] && [ $RELEASE == "stretch" ]; then
+  sudo lxc-attach -n "$NAME" -- apt install sudo
+
```

Comment:
What happens if we install the `sudo` package and it's already installed? Maybe we can avoid the `if`.

Reply:
You are right, we can avoid the `if`:

```
root@tryton:/home/tryton# apt install sudo
Reading package lists... Done
Building dependency tree       
Reading state information... Done
sudo is already the newest version (1.8.19p1-2.1).
0 upgraded, 0 newly installed, 0 to remove and 22 not upgraded.
```